catalog/core/catalog-core-subscriptionstore/subscriptionstore-impl/pom.xml

```diff
@@ -52,6 +72,17 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Import-Package>
+                            !javax.enterprise.util,
+                            !org.xmlpull.v1,
+                            *
+                        </Import-Package>
+                        <Embed-Dependency>
+                            cache2k-all,
+                            cache2k-jcache,
+                            cache2k-api,
+                            cache2k-jcache-api
+                        </Embed-Dependency>
```

Comment:
I could be wrong but it looks like your exporting all of these classes are you okay with that and if we need to be exporting things should you move the interfaces to a separate bundle so that implementors aren't dependent on chache2k.